features/admin-file-attributes.feature

```diff
@@ -0,0 +1,42 @@
+@admin-file @issue-457
+  Feature: A curator can manage file attributes in admin file update page
+    As a curator,
+    I want to manage file attributes from the update form
+    So that I can associate various attributes to files
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+
+
+  @ok @issue-457 @javascript
```

Comment:
This scenario does not involve javascript-enabled features so does require the @javascript tag to pass?

Reply:
@javascript tag removed and passed the test.